css/app.css

```diff
@@ -1048,6 +1071,9 @@ button.save.has-count .count::before {
     border-radius: 0 0 4px 4px;
     overflow: hidden;
 }
+.form-field > input:nth-child(3) {
+    border-left: 0;
+}
```

Comment:
Because the ui.type.range needs second input without border. 
Would you suggest a better selector maybe?